src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -717,6 +718,25 @@ def load_arguments(self, _):
     with self.argument_context('aks mesh upgrade start') as c:
         c.argument('revision', validator=validate_azure_service_mesh_revision, required=True)
 
+    with self.argument_context("aks approuting enable") as c:
+        c.argument("enable_kv", action="store_true")
+        c.argument("keyvault_id", options_list=["--attach-kv"])
+
+    with self.argument_context("aks approuting update") as c:
+        c.argument("keyvault_id", options_list=["--attach-kv"])
+        c.argument("enable_kv", action="store_true")
+
+    with self.argument_context("aks approuting zone add") as c:
+        c.argument("dns_zone_resource_ids", options_list=["--ids"], required=True)
+        c.argument("attach_zones")
+
+    with self.argument_context("aks approuting zone delete") as c:
+        c.argument("dns_zone_resource_ids", options_list=["--ids"], required=True)
+
+    with self.argument_context("aks approuting zone update") as c:
+        c.argument("dns_zone_resource_ids", options_list=["--ids"], required=True)
+        c.argument("attach_zones")
```

Comment:
After some investigation, I found out that the reason why Check CLI Linter fails is due to the use of double quotes here. Please use single quote instead to pass CI.

Created an issue to the tool [#434](https://github.com/Azure/azure-cli-dev-tools/issues/434).

Reply:
updated to single quote